notify/api/v1/api.py

```diff
@@ -91,8 +91,8 @@ def send_notification(backends):
                 result["result"][backend][drv_name] = {"error": str(e)}
                 result["errors"] += 1
             except Exception as e:
-                LOG.error(("Backend '{}' driver '{}' "
-                           "error: {}").format(backend, drv_name, e))
+                LOG.error("Backend '{}' driver '{}': {}: {}".format(
```

Comment:
I changed all such occurrences in seecloud/availability#24 and other projects as well.